### DISCUSSION TITLE
Salt Stripping

### DISCUSSION THREAD

**Discussion by pwging13** [rdkit/rdkit#6574] [Q&A]
I need to salt/solvent strip a bunch of molecules.  I currently use KNIME and the RDKit Salt Stripper node where I pass a custom list of salts/solvents/fragments to remove.

I'd like to move away from knime and implement this in python.

Does one use rdkit.Chem.MolStandardize.rdMolStandardize.FragmentRemoverFromData((str)fragmentData[, (bool)leave_last=True[, (bool)skip_if_all_match=False]])?  If so, what's the format desired for the fragment data string?

Or does one use rdkit.Chem.MolStandardize.rdMolStandardize.RemoveFragments((Mol)mol[, (AtomPairsParameters)params=None]) and pass cleanup params, setting the fragmentFile property to a custom file.  If so, what's the format of that fragment file?

Thanks

**Comment by DrewG** [comment]
Hi, have a look at the SaltRemover module - https://www.rdkit.org/docs/source/rdkit.Chem.SaltRemover.html
